CompatBot/Commands/Misc.cs

```diff
@@ -90,8 +90,8 @@ public async Task Credits(CommandContext ctx)
                     Url = "https://github.com/RPCS3/discord-bot",
                     Color = DiscordColor.Purple,
                 }.AddField("Made by",
-                    "🇭🇷 Roberto Anić Banić aka nicba1010\n" +
-                    "💮 13xforever")
+                    "💮 13xforever" +
+                    "🇭🇷 Roberto Anić Banić aka nicba1010\n")
```

Comment:
This may or may not result in an empty line that could break the formatting, depending on whether discord likes it or not in current update :(
```suggestion
                    "🇭🇷 Roberto Anić Banić aka nicba1010")
```